spec/models/pieces/king_spec.rb

```diff
@@ -1,4 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe King, type: :model do
+  let(:game) { FactoryGirl.create(:game) }
+  Piece.destroy_all
```

Comment:
I don't see where you need `game`. Or `Piece.destroy_all`

Everything else looks good though, feel free to merge yourself when you're ready 😄